mcs/packages/Makefile

```diff
@@ -3,7 +3,7 @@ include ../build/rules.make
 
 ROSLYN_CSC_DIR = $(dir $(CSC_LOCATION))
 
-ROSLYN_FILES = \
+ROSLYN_FILES_FOR_MONO = \
```

Comment:
re:csi, Yeah, I'll install that to `4.5` and add the script.

I was about talking `csi` being surfaced as a msbuild task, but it looks like it isn't actually referenced in the targets files at all! So, it doesn't have a corresponding `$(CsiToolPath)`, which means that we should have `csi.exe` available (symlinked) in the msbuild directory. And with that the `*Scripting*` assemblies will need to be linked too.

`VBCSCompiler` is used by the `Csc` task, which would expect it to be in the msbuild directory. So, I could install+symlink this.
Another thing to note here though - this isn't usable right now as Roslyn disables it's use (https://github.com/dotnet/roslyn/blob/master/src/Compilers/Shared/BuildServerConnection.cs#L68).